src/test/java/uk/nhs/hee/tis/trainee/reference/service/LocalOfficeContactTypeServiceTest.java

```diff
@@ -116,6 +122,8 @@ void createLocalOfficeContactTypeShouldUpdateLocalOfficeContactTypeWhenExistingT
 
     LocalOfficeContactType contactType = service.create(contactType2);
 
+    verify(contactService, atLeastOnce()).updateAllForContactType(contactType1);
```

Comment:
It actually updates them twice, since the 'create' calls the 'update'. Not so pretty :/